### PR TITLE
userProfile

### PR DESCRIPTION
**Pull request by tulasikurete**
image

### CODE REVIEW

**Comment by tulasikurete** [comment]
image
image
image

File: src/app/layout.tsx [hunk 1]
```diff
@@ -43,7 +46,10 @@ export default function RootLayout({
             </Flex>
           </Header>
           <Layout>
-            <Content style={contentStyle}>{children}</Content>
+            <Content style={contentStyle}>
+              {children}
```
Comment (arajajyothibabu) [inline src/app/layout.tsx]: Remove changes in this file

File: src/app/globals.css [hunk 2]
```diff
@@ -119,3 +119,4 @@ a {
   height: 150px;
   border: 1px dashed grey;
 }
+
```
Comment (arajajyothibabu) [inline src/app/globals.css]: revert this change

File: prisma/schema.prisma [hunk 3]
```diff
@@ -76,3 +76,4 @@ model Order {
   address           Address
 }
 
+
```
Comment (arajajyothibabu) [inline prisma/schema.prisma]: revert this change

**Review by arajajyothibabu — CHANGES_REQUESTED** [review]

**Comment by tulasikurete** [comment]
image

**Review by arajajyothibabu — APPROVED** [review]

**Pull request merged**